src/RedisClusterOptions.php

```diff
@@ -64,6 +64,8 @@ final class RedisClusterOptions extends AdapterOptions
     /** @psalm-var array<positive-int,mixed> */
     private array $libOptions = [];
 
+    private ?string $username = null;
```

Comment:
Please persist either `null` or `non-empty-string`. Or simply use `string` and declare empty as "non-existent". Same same but that prevents three states which needs to be checked.

Reply:
I will go then with the second option to be consistent with the password in RedisClusterOptions